src/app/app.component.ts

```diff
@@ -1,15 +1,27 @@
-import { Component } from '@angular/core';
-
+import { Component, OnInit } from '@angular/core';
+import { Broadcaster } from './shared/broadcaster.service';
 import { AuthenticationService } from './auth/authentication.service';
 
 @Component({
   selector: 'alm-app',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
-  })
-export class AppComponent {
+})
+export class AppComponent implements OnInit {
   
-  constructor(auth: AuthenticationService){
+  constructor(private auth: AuthenticationService, private broadcaster: Broadcaster){
     //auth.isLoggedIn();
   }
-}
\ No newline at end of file
+  loggedIn: Boolean = false;
```

Comment:
What are you trying to do here? How does this relate to fixing the padding for login?

Usually the app.component.ts file is for bootstrapping the entire application. So this kind of thing is not written here.